README.md

```diff
@@ -2,31 +2,50 @@
 
 An Alpine Linux based docker image for [GoCD server](https://www.gocd.io).
 
-# Usage
+# Docker Image
+*You could either download the latest built image from docker hub or build one
+locally with the zip file from GoCD Download webpage.*
 
-Start the container with this:
+## 1.1 Download the latest Image
+```bash
+docker pull gocd/gocd-server:latest
```

Comment:
We do not publish the `latest`. Our tags on dockerhub will be of the format `v<version>` where version refers to the GoCD version, like this `v17.3.0`